pom.xml

```diff
@@ -64,7 +64,7 @@
 
 		<!-- redefine versions of some libraries defined by Spring platform -->
 		<postgresql.version>42.0.0</postgresql.version>
-		<tomcat.version>7.0.77</tomcat.version>
+		<tomcat.version>7.0.56</tomcat.version>
```

Comment:
The really used version of Tomcat on most instances is 7.0.56, not the latest. It probably has no influence, as the difference in Tomcat API among patchlevels is zero, but just to be on the safe side, the really used version is set here.